starknet_py/tests/e2e/conftest.py

```diff
@@ -469,6 +471,112 @@ async def cairo_serializer(gateway_account_client: AccountClient) -> CairoSerial
     return CairoSerializer(identifier_manager=contract.data.identifier_manager)
 
 
+@pytest_asyncio.fixture(scope="module")
+async def deployer_address(gateway_client: AccountClient) -> int:
+    """
+    Returns an address of the UDC
+    """
+    deploy_tx = make_deploy_tx(
+        compilation_source=(mock_dir / "contracts/universal_deployer.cairo").read_text(
+            "utf-8"
+        )
+    )
+    res = await gateway_client.deploy(deploy_tx)
```

Comment:
For now, it uses an old 'Deploy` transaction, but there is an issue in devnet to create pre-deployed UDC, so we will be able to replace this code.